x-pack/plugins/security_solution/public/management/components/artifact_list_page/artifact_page.tsx

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback, useMemo } from 'react';
+import styled from 'styled-components';
+
+import type { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+import { EuiButton, EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import { EuiFlyoutSize } from '@elastic/eui/src/components/flyout/flyout';
+import { AdministrationListPage as _AdministrationListPage } from '../administration_list_page';
+
+import { PaginatedContent } from '../paginated_content';
+
+import { ArtifactEntryCard } from '../artifact_entry_card';
+
+import { MANAGEMENT_DEFAULT_PAGE_SIZE, MANAGEMENT_PAGE_SIZE_OPTIONS } from '../../common/constants';
+import { ArtifactListPageLabels, artifactListPageLabels } from './translations';
+import { useTestIdGenerator } from '../hooks/use_test_id_generator';
+import { ManagementPageLoader } from '../management_page_loader';
+import { SearchExceptions } from '../search_exceptions';
+import { useArtifactCardPropsProvider } from './hooks/use_artifact_card_props_provider';
+import { NoDataEmptyState } from './components/no_data_empty_state';
+import { ArtifactFlyoutProps, MaybeArtifactFlyout } from './components/artifact_flyout';
+import { useIsFlyoutOpened } from './hooks/use_is_flyout_opened';
+import { useSetUrlParams } from './hooks/use_set_url_params';
+import { useFetchArtifactData } from './hooks/use_fetch_artifact_data';
+
+type ArtifactEntryCardType = typeof ArtifactEntryCard;
+
+const AdministrationListPage = styled(_AdministrationListPage)`
+  // TODO:PT Ask David - why do we have this here? because the Card already has similar code:
```

Comment:
Good point. I don't really know it, it was introduced [here](https://github.com/elastic/kibana/pull/98675/files#diff-7ebca7d3befeba3652d93b7ef771963b64006f2fccb16812cf943752c2a153b8R38) so I assume yes, we can move it

Reply:
cool. Thanks. I will do that.